src/wp-includes/class-wp-script-modules.php

```diff
@@ -225,6 +226,32 @@ public function print_import_map() {
 		}
 	}
 
+	/**
+	 * Prints the necessary script to load import map polyfill for browsers that
+	 * do not support import maps.
+	 *
+	 * @since 6.5.0
+	 */
```

Comment:
The global used should be included in PHPDoc as explained in https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods.

Reply:
Done!